src/ast/passes/field_analyser.h

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
-#include "bpftrace.h"
-#include "visitors.h"
```

Comment:
Just highlighting an example of where the include path was neither truly relative to the file itself nor to the project source root, in case it helps the review.

`field_analyser` is in `src/ast/passes`, but the included `visitors.h` is in `src/ast`